js/ui/collection/ui.collection_widget.base.js

```diff
@@ -973,10 +973,15 @@ const CollectionWidget = Widget.inherit({
         return itemTemplate.render({
             model: renderArgs.itemData,
             container: renderArgs.container,
-            index: renderArgs.index
+            index: renderArgs.index,
+            onRendered: this._onItemTemplateRendered(itemTemplate, renderArgs)
         });
     },
 
+    _onItemTemplateRendered: function() {
```

Comment:
Seems like it should looks like 
```suggestion
    _onItemTemplateRendered: noop,
```

Reply:
it needs to return noop to be callable cause we have to pass render args to closure in widgets async